docs/README.md

```diff
@@ -32,52 +32,8 @@ $ pip install anta
 
 # Run ANTA CLI
 $ anta --help
-Usage: anta [OPTIONS] COMMAND [ARGS]...
-
-  Arista Network Test Automation (ANTA) CLI
-
-Options:
-  --version                       Show the version and exit.
-  --username TEXT                 Username to connect to EOS  [env var:
-                                  ANTA_USERNAME; required]
-  --password TEXT                 Password to connect to EOS that must be
-                                  provided. It can be prompted using '--
-                                  prompt' option.  [env var: ANTA_PASSWORD]
-  --enable-password TEXT          Password to access EOS Privileged EXEC mode.
-                                  It can be prompted using '--prompt' option.
-                                  Requires '--enable' option.  [env var:
-                                  ANTA_ENABLE_PASSWORD]
-  --enable                        Some commands may require EOS Privileged
-                                  EXEC mode. This option tries to access this
-                                  mode before sending a command to the device.
-                                  [env var: ANTA_ENABLE]
-  -P, --prompt                    Prompt for passwords if they are not
-                                  provided.
-  --timeout INTEGER               Global connection timeout  [env var:
-                                  ANTA_TIMEOUT; default: 30]
-  --insecure                      Disable SSH Host Key validation  [env var:
-                                  ANTA_INSECURE]
-  -i, --inventory FILE            Path to the inventory YAML file  [env var:
-                                  ANTA_INVENTORY; required]
-  --log-file FILE                 Send the logs to a file. If logging level is
-                                  DEBUG, only INFO or higher will be sent to
-                                  stdout.  [env var: ANTA_LOG_FILE]
-  --log-level, --log [CRITICAL|ERROR|WARNING|INFO|DEBUG]
-                                  ANTA logging level  [env var:
-                                  ANTA_LOG_LEVEL; default: INFO]
-  --ignore-status                 Always exit with success  [env var:
-                                  ANTA_IGNORE_STATUS]
-  --ignore-error                  Only report failures and not errors  [env
-                                  var: ANTA_IGNORE_ERROR]
-  --help                          Show this message and exit.
-
-Commands:
-  debug  Debug commands for building ANTA
-  exec   Execute commands to inventory devices
-  get    Get data from/to ANTA
-  nrfu   Run NRFU against inventory devices
+--8<-- "anta_help.txt"
```

Comment:
we removed this because it does not render well on Github sadly ...so need to put the full command there .